transport_nantes/asso_tn/templates/asso_tn/base_mobilitain.html

```diff
@@ -103,11 +112,27 @@
 			if they are designated with a string.
 			{% endcomment %}
 			{% if is_static %}
-				<meta name="og:image" content="{% static social.og_image %}" />
+				{# OG image #}
+				{% if social.og_image and social.og_image != "velopolitain/v1.png" %}
+					<meta property="og:image" content="{{social.og_image.url}}" />
+				{% elif social.twitter_image and social.twitter_image != "asso_tn/mobilite-pour-tous.jpg"%}
```

Comment:
Here and later, why are you comparing to this hard-coded fixed string?  The base template has no knowledge of what or why any particular image file would be supplied.

Reply:
When I try everything empty i still got some " default reponse " this is why I check if this is not the " default " the default are in This file  tn\tn_web\transport_nantes\asso_tn\middleware\default_context.py